### DISCUSSION TITLE
My page deploys as a blank

### DISCUSSION THREAD

**Discussion by coffeeprynce** [community/community#51530] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Hi I am creating a portfolio app using vite and three js and it works fine locally but when i went to deploy it on github pages it just shows up as blank can someone help me please.

this is my repo link https://github.com/coffeeprynce/3d-portfolio

**Comment by iaruso** [comment]
Hi, I just tried one thing and it worked for me.

As provided in the documentation:
`import { Tilt } from 'react-tilt'`

You just need to add { } around Tilt import statement in Works.jsx and About.jsx.

**Reply by coffeeprynce** [reply]
so i tried what you suggested and its still blank was there anything else you did to it or was that it?

**Comment by radar-blip** [comment]
In your index.html, the path in this link seems to be wrong. 

Try changing this:
`<script type="module" src="/src/main.jsx"></script>`

to this:
`<script type="module" src="src/main.jsx"></script>`

**Reply by coffeeprynce** [reply]
I tried that but its still showing up as blank unfortunately =(

**Comment by iaruso** [comment]
The 'react-parallax-tilt' package you added works for me.

But answering to the main question. with 'react-tilt' I got this error:
`Uncaught SyntaxError: ambiguous indirect export: default`
I just added the curly braces to resolve. 
Also, I had other errors that I can't replicate now. When I did "npm install", there were incompatibilities between the 'react' and 'react-tilt' packages.

Edit: show the error you getting when you deploy.

**Reply by coffeeprynce** [reply]
so the error you are getting could possibly because you have to use legacy peer deps when installing packages just because some packages are not compatible with each other because of deprecation. also are you deploying this to github or just running it locally? because locally it works perfectly fine for me i'm not getting any errors at all, its when i deploy to github pages that its showing up blank again no errors are showing up in console its just showing a blank white page

**Reply by iaruso** [reply, marked as answer]
I had to test a lot to fix it, but here it goes:

1. Since your `npm run build` creates a dist folder, I changed from your `npm run deploy` to this `"deploy": "gh-pages -d dist"`.

2. The problem about being blank are the references, in your `index.html` and javascript file from dist folder, you just needed a period when referencing assets, so I modified your `vite.config.js` file to this:

```
import { defineConfig } from 'vite';
import react from '@vitejs/plugin-react';
import viteCompression from 'vite-plugin-compression';

// https://vitejs.dev/config/
export default defineConfig(({ command, mode }) => {
  return {
    plugins: [
      react(),
      viteCompression()
    ],
    assetsInclude: ['**/*.jpg', '**/*.png', '**/*.svg', '**/*.gif'],
    base: './'
  }
});
```

3. After this just `npm run build`, then `npm run deploy` and wait.

Hope this finally resolved your issue :).

**Reply by coffeeprynce** [reply]
omg yes that finally did it! thank you so much for your patience and all your help I had tried about a thousand things before this, I really appreciate your help and all credits for debugging it goes to you

**Reply by IggyMartin** [reply]
That same thing help me too. Thanks a lot

**Comment by Bebavaldezbryt** [comment]
No puedo administrar mi pagos

**Comment by EagleSeer** [comment]
Hello i have the same issue, but i'm using react +typescript. I really need please 
this is my repo  link https://github.com/EagleSeer/Photoshop
